webui/src/pages/extension-detail/extension-review-dialog.tsx

```diff
@@ -107,7 +122,9 @@ class ExtensionReviewDialogComponent extends React.Component<ExtensionReviewDial
                     <DialogContentText>
                         Your review will be posted publicly as {this.props.user.loginName}
                     </DialogContentText>
-                    <ExtensionRatingStarSetter ref={ref => this.starSetter = ref} />
+                    <div className={this.props.classes.stars}>
```

Comment:
Shouldn't this be done in the ExtensionRatingStarSetter component? There you could set the size of `IconButton` to `'small'` instead of scaling afterwards, or reduce the spacing between stars to avoid the line break.

Reply:
No, let's keep your current approach. Even if it's not optimal, it solves the original issue and keeps the site unchanged for large devices.

Reply:
Ok. Thanks